app/server/src/generator/templates/template1/index.js

```diff
@@ -354,6 +367,9 @@ function template1(values: SanitizedValues) {
           case 'profile':
             return generator.profileSection(values.basics)
 
+          case 'about':
+            return generator.aboutSection(values.basics, headings.about)
```

Comment:
Because we want to set the `aboutSection` method in the `Generator` type to optional for now, this will have a type error because Flow won't be sure that the `aboutSection` method exists or not. For now, we can just add a check to make sure it exists before using it to get rid of the type error (we can remove this method check at some point once we have the `aboutSection` implemented in every template)

```suggestion
              return generator.aboutSection ? generator.aboutSection(values.basics, headings.about) : ''
```